pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -766,6 +767,10 @@ private String composeTableGaugeName(final String tableName, final String key, f
     return gauge.getGaugeName() + "." + getTableName(tableName) + "." + key;
   }
 
+  public String composePluginGaugeName(String pluginName, Gauge gauge) {
+    return pluginName + StringUtils.capitalize(gauge.getGaugeName());
```

Comment:
do you want to add a dot int between ?
`pluginName + "." + StringUtils.capitalize(gauge.getGaugeName());`

Reply:
Makes sense, will do.